mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -246,36 +246,27 @@ export function DexScreen (): JSX.Element {
       <TopLiquiditySection onPress={onSwap} pairs={topLiquidityPairs} />
       <NewPoolsSection onPress={onAdd} pairs={newPoolsPairs} />
       <View style={tailwind('flex-1')}>
-        {activeTab === TabKey.AvailablePoolPair &&
-          (!hasFetchedPoolpairData || isSearching) && (
+        {activeTab === TabKey.AvailablePoolPair && (!hasFetchedPoolpairData || isSearching) &&
+          (
             <ThemedScrollView contentContainerStyle={tailwind('p-4')}>
               <SkeletonLoader row={4} screen={SkeletonLoaderScreen.Dex} />
             </ThemedScrollView>
-          )}
-        {activeTab === TabKey.AvailablePoolPair &&
-          hasFetchedPoolpairData &&
-          !isSearching && (
-            <PoolPairCards
-              availablePairs={filteredAvailablePairs}
-              yourPairs={yourLPTokens}
-              onAdd={onAdd}
-              onRemove={onRemove}
-              onSwap={onSwap}
-              type='available'
-              setIsSearching={setIsSearching}
-              searchString={searchString}
-              buttonGroupOptions={{
```

Comment:
can we revert the removal of `buttonGroupOptions` since this PR only updating the UI of available pool pair